src/site/static/eligibility.liquid

```diff
@@ -420,6 +420,7 @@ pageClass: "page-public-assistance"
         liheap: Low Income Home Energy Assistance Program (LIHEAP)
         medical: Medi-Cal <span class="subtitle">Also known as California’s Medicaid program</span>
         nslp: National School Lunch Program (NSLP)
+        schip: State Children's Health Insurance Program (SCHIP) <span class="subtitle">Also known as Healthy Families A & B</span>
         ssi: Supplemental Security Income (SSI)
```

Comment:
I just noticed I did not properly alphabetize this list in the first place, doh!  >.<

Your insertion looks good though.  I'll fix the ordering of the other items later.